hudi-spark-datasource/hudi-spark4-common/src/main/scala/org/apache/spark/sql/adapter/BaseSpark4Adapter.scala

```diff
@@ -73,6 +73,8 @@ abstract class BaseSpark4Adapter extends SparkAdapter with Logging {
 
   override def getAvroSchemaConverters: HoodieAvroSchemaConverters = HoodieSparkAvroSchemaConverters
 
+  override def getHoodieSchemaConverters: HoodieSchemaConverters = HoodieSparkSchemaConverters
```

Comment:
It looks like all Spark versions use the same `HoodieSparkAvroSchemaConverters` class now.  So there is no need to have `getAvroSchemaConverters` or `getHoodieSchemaConverters`, i.e., `HoodieSparkSchemaConverters` can be directly used.  Does that work or the adapter method has to be created?